x/subkeys/internal/types/expiration.go

```diff
@@ -0,0 +1,107 @@
+package types
+
+import "time"
+
+// ExpiresAt is a point in time where something expires.
+// It may be *either* block time or block height
+type ExpiresAt struct {
```

Comment:
See the above comment. Seems like we can really simplify expiration handling by just using int64

Reply:
See comment above. We need to be very explicit if we are setting time or height. The fact that we measure two things with the same type doesn't mean they are interchangeable.

But I think I am missing something here.

Reply:
See above. Let's use int64 at least. Resolving.